components/d2l-organization-consortium/lang/fr-on.js

```diff
@@ -0,0 +1,9 @@
+export default {
+	"errorFull": "Oups! Nous avons été incapables d\'extraire l\'information de {num} de vos onglets. Veuillez rafraîchir la page",
+	"errorShort": "Oups!",
+	"impersonationWarning": "Vous ne pouvez pas accéder à d\'autres onglets lorsque vous personnifiez un autre utilisateur.",
```

Comment:
```suggestion
	"errorFull": "Oups! Nous avons été incapables d'extraire l'information de {num} de vos onglets. Veuillez rafraîchir la page",
	"errorShort": "Oups!",
	"impersonationWarning": "Vous ne pouvez pas accéder à d'autres onglets lorsque vous personnifiez un autre utilisateur.",
```
Similar to last time - you can remove these now, but they shouldn't hurt anything either way